randovania/games/prime2/logic_database/Torvus Bog.txt

```diff
@@ -1359,6 +1359,8 @@ Extra - in_dark_aether: False
               Any of the following:
                   Movement (Expert)
                   Morph Ball Bomb and Spider Ball
+          # https://www.youtube.com/watch?v=pUYX81KhsGg
+          Morph Ball Bomb and Boost Ball and Morph Ball and Space Jump Boots and Bomb Space Jump (Advanced) and Movement (Expert) and Standable Terrain (Advanced)
```

Comment:
I kinda think the BSJ should be bumped to Expert mainly because it also involves balancing on the branch.

Reply:
I don't mind changing this to Expert, considering that you need to ride the wall behind you to get enough height as well